### ISSUE TITLE
nix search --verbose does not change anything

### ISSUE THREAD

**Issue by guibou** [NixOS/nix#2473]
The output of `nix search` and `nix search --verbose` are the same:

```
$ nix search blender
warning: using cached results; pass '-u' to update the cache
* nixpkgs.blender (blender)
  3D Creation/Animation/Publishing System
$ nix search --verbose blender
warning: using cached results; pass '-u' to update the cache
* nixpkgs.blender (blender)
  3D Creation/Animation/Publishing System
```

I was actually hoping that `--verbose` would display version number, but no. `--json` can be used as a workaround.

**Comment by unode** [comment]
Also stumbled on the same issue.
Seems like [nixs](https://github.com/unode/nixs/) still has one use-case remaining.

**Comment by timor** [comment]
At the moment (nix 2.2.2) it seems that `nix search` also emits the package version, so there again is no difference.  I think it would be useful if the `--verbose` flag shows the homepage information of a package.

**Comment by guibou** [comment]
@timor using nix 2.2.2 I have no version in the default output.

**Comment by timor** [comment]
@guibou Right.  Seems I mistook some version component in the attribute name for the version info.

Please disregard, then.

**Comment by tomberek** [comment]
Closing as stale. Re-open if needed.
